test/controllers/apply_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
You can add a check for responds to get just so it's not empty.  There are examples in the other test files.  There should be a test for that for every page.

Reply:
I didn't put anything here because we haven't really started working on the apply part yet.

Reply:
If the page exists it should have a test to prove it.

Reply:
put some existence tests